src/js/components/ConnectionModal/StatusLight.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import styled from "styled-components"
+import {ClusterStatus} from "../../state/Clusters/types"
+
+const StatusLight = ({...props}: any) => {
+  return (
+    <svg {...props} viewBox="0 0 100 100" width="10">
+      <defs>
+        <filter id="f1" x="-50%" y="-50%" width="200%" height="200%">
+          <feGaussianBlur in="SourceGraphic" stdDeviation="5" />
+        </filter>
+      </defs>
+      <circle cx="50" cy="50" r="40" strokeWidth="3" filter="url(#f1)" />
+    </svg>
```

Comment:
If you didn't want to mess with svgs, I think this could be done using css.

pseudo-css
```
width: 10px;
height: 10px;
border-radius: 50%;
background: $color;
box-shadow: 0 0 5px 3px transparentize($color, a little bit);
```